kythe/go/serving/xrefs/xrefs.go

```diff
@@ -55,8 +57,15 @@ import (
 
 var (
 	mergeCrossReferences = flag.Bool("merge_cross_references", true, "Whether to merge nodes when responding to a CrossReferencesRequest")
+
+	experimentalCrossReferenceIndirectionKinds flagutil.StringMultimap
 )
 
+func init() {
+	flag.Var(&experimentalCrossReferenceIndirectionKinds, "experimental_cross_reference_indirection_kinds",
+		`Comma-separated set of key-value pairs (node_kind=edge_kind) to indirect through in CrossReferences.  For example, "talias=/kythe/edge/aliases" indicates that the targets of a 'talias' node's '/kythe/edge/aliases' related nodes will have their cross-references merged into the root 'talias' node's.`)
```

Comment:
It looks like it will, but does this work with reverse edges? I think we want to follow `%aliases` for the aliases-for-import case (e.g., in `from foo import fn as gn`, `gn aliases fn`, and xrefs for `fn` should be a superset of the xrefs of `gn`).

Reply:
It will work for any edge kind, reverse or not.